apps/studio/csp.js

```diff
@@ -26,6 +26,11 @@ const SUPABASE_CONTENT_API_URL = process.env.NEXT_PUBLIC_CONTENT_API_URL
   ? new URL(process.env.NEXT_PUBLIC_CONTENT_API_URL).origin
   : ''
 
+const isDevOrStaging =
+  process.env.NEXT_PUBLIC_VERCEL_ENV === 'preview' ||
+  process.env.NEXT_PUBLIC_ENVIRONMENT === 'local' ||
+  process.env.NEXT_PUBLIC_ENVIRONMENT === 'staging'
```

Comment:
just a nit but wondering if it might be better to check for isProduction? so that all other values will default to use `ph.supabase.green` instead (if for any reason the env value messes up then we won't affect prod PostHog)

`const isProduction` = process.env.NEXT_PUBLIC_ENVIRONMENT === 'prod'`